### DISCUSSION TITLE
How to run hundreds of thousands of jobs per month for cheap?

### DISCUSSION THREAD

**Discussion by lancejpollard** [quirrel-dev/quirrel#718] [Ideas]
What is as cheap as you can get in theory if you were to do this on AWS directly? How would it be charged? Could you offer it for just slightly more than that price 😅 ?

It would be cool to hear reasons why you choose to [charge](https://quirrel.dev/#pricing) $20/10k calls, it seems like charging per hour would make more sense.

Just trying to figure out what it would cost at a bare minimum so I could do background jobs, but then find a nice service like this for a slightly higher premium. It looks great so far, but only having 100 jobs for free will be used up just in testing and getting it working, and $20/mo for an app without any users to start is a lot.

Sidenote, this is for a side project app if that makes any difference, maybe you have discounted pricing :)

Thanks for reading!

**Comment by Skn0tt** [comment]
Hi @lancejpollard! Quirrel does not execute your jobs, but orchestrates & triggers them. The actual execution of your job is performend on your application server, as usual. That's why per-hour pricing isn't a great fit.

> 100 jobs for free will be used up in testing

That's why you can use the Quirrel CLI to test locally! :)

> $20/mo for an app without any users to start is a lot. this is for a side project app

I do offer sideproject discounts, please [drop me an email](mailto:info@quirrel.dev?subject=OSS%20%2F%20Side%20Project%20Discount&body=Hi%20there!%0A%0AI'd%20like%20to%20apply%20for%20Quirrel's%20OSS%20%2F%20Side%20Project%20discount.%0A%0AHere's%20a%20link%20to%20my%20project%3A%20%5BLink%5D%0AThis%20is%20what%20I%20want%20to%20use%20Quirrel%20for%3A%20%5B...%5D%0AI%20expect%20to%20make%20%5B...%5D%20monhtly%20Quirrel%20calls.%0A%0ALooking%20forward%20to%20you%20reply!%0A%0ABest%2C%0A...)!
